docs/documentation/index.html

```diff
@@ -1457,6 +1457,18 @@ <h5 class="m-documentation--configuration-side-title">Core configuration</h5>
           Check if SSL certificate is valid <br>
           <span class="tag">Default:</span> <code>true</code>
         </dd>
+
+        <dt><code>MO_MONITORABLE_HTTP_CERTIFICATE</code> <code class="type">string</code></dt>
+        <dd>
+          Cient certificate file path to use when making http calls <br>
+          <span class="tag">Default:</span> <code></code>
```

Comment:
So there is no default?

Reply:
Default is not certificate. It's more an optional value than a defaulted value.
Would this change express it nicely ?